neptune-python-utils/readme.md

```diff
@@ -80,6 +80,20 @@ from neptune_python_utils.endpoints import Endpoints
 endpoints = Endpoints(role_arn='arn:aws:iam::...')
 ```
 
+To use it in a Secure Network Environment:
+
+Suppose your service handles critical PII data, so you have configured your VPC to not allow internet access by having no PublicSubnets and NAT Gateways in your VPC. Resources in such a VPC won't even have access to AWS services' endpoints. To access AWS services, you'd have to add InterfaceEndpoints or GatewayEndpoints of specific services in your VPC.   
```

Comment:
-- rephrase to: If your Amazon VPC configuration doesn't have a public subnet,   AWS services' endpoints like STS can be accessed via VPC endpoints (InterfaceEndpoints or GatewayEndpoints). AWS STS has regional endpoints, listed here - **Using AWS STS interface VPC endpoints** <Convert this to Link>

To enable the library to successfully connect to STS, pass STS regional endpoint value in `endpoint_url` parameter.

-- remove ref to Pull Request, the doc doesn't have reference to other pull requests

Reply:
addressed